hey/cli.py

```diff
@@ -74,3 +76,13 @@ def ask(
 
     result = answer(user_input, no_style)
     console.print(result)
+
+@app.command()
+def auth(
+    user_input: str, # This for unhandled exceptions that happen when user gives too many args.
+):
+    """
+    Take HEY_TOKEN from user.
+    """
+    password = getpass.getpass("Copy and paste your token here, [Token will not be echoed] > ")
+    keyring.set_password("system","HEY_TOKEN",password)
```

Comment:
I don't seem to understand your request, why should we store a keyring value as a constant in `hey/consts.py` ?

Reply:
@sepehrrasooli Soon, there might be a situation where we need to change the word `HEY_TOKEN` to something else. Making it dynamically set as a constant variable would allow us to have full control over it.

Reply:
I get it, I will fix these problems ASAP.

Reply:
<3